components/shared/LinkButton.js

```diff
@@ -66,22 +69,57 @@ const LinkButton = ({
     });
   };
 
+  const OutlineLink = () => {
+    return (
+      <StyledOutlineLink
+        href={href}
```

Comment:
With `href` set here for the anchor tag, will next follow this `href` or will `Link` override it? Wondering which behavior it will follow with the `href` in both essentially.

Reply:
It works as expected. Both href values should be same as they are both based on the same property. Am I missing something?

Reply:
The `a` tag will do a page refresh, where `Link` gets a bit fancier and only loads the page portion, with the header and footer staying in tact. Was wondering which one won when both have an `href` set.

Reply:
I poked around a bit and still seems to act like a `Link` 👍